### PR TITLE
SCIF + NWIF badge alerts and role updates

### PR DESCRIPTION
**Pull request by saralain88**
This updates the bot to support scif badges as well as nwif. (Discord posts + user roles) This admittedly isn't my most clean code, but I also feel like having _something_ sooner rather than later is probably good. Feedback is of course welcome!

When releasing, we'll also need to change some environment variables. Changes needed: 

* Add NODE_ENV=production variable (if not already present/inferred somehow)
* Rename all variables specific to NWIF with postfix _NWIF (airtable stuff, badge channels)
* Add same variables for scif with postfix _SCIF
* Update AIRTABLE_BADGE_MAP to prefix all NWIF badges with "NWIF " 
* Update AIRTABLE_BADGE_MAP with new values with "SCIF" prefix

### CODE REVIEW

**Review by shirgoldbird — APPROVED** [review]
LGTM! It's extensible to lots of different events which is great, and I think appending the event name (vs prepending) is the right call so it logically groups variables.

File: lib/airtable.js [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 
-const getBase = async () => {
-	const client = await getClient
-	const base = client.base(AirtableOptions.baseID)
+const getClient = async (options) => {
+	if (existingClients[options.baseID]) {
+		return existingClients[options.baseID]
+	}
+	const client = new Airtable({ apiKey: options.apiToken })
+	existingClients[options.BaseID] = client
```
Comment (stevestreza) [inline lib/airtable.js:11]: I think this should be `options.baseID`, not `options.BaseID`

File: lib/options.js [hunk 2]
```diff
@@ -17,35 +17,49 @@ const discordAirtableBadgeMap = assertToken(
...
+	apiToken: airtableAPITokenNwif,
+	baseID: airtableBaseIDNwif,
+}
+const AirtableScif = {
+	apiToken: airtableAPITokenScif,
+	baseID: airtableBaseIDScif,
 }
 
 const scheduleUrl = assertToken(`SCHEDULE_URL`)
+const nodeEnv = assertToken(`NODE_ENV`)
```
Comment (stevestreza) [inline lib/options.js:47]: Doesn't look like this gets used

**Review by stevestreza — COMMENTED** [review]
This pattern is fine for now but there's a lot of duplication, it probably makes more sense to have an object containing all of the properties (discord ID, airtable ID/token/instance) and having separate instances for NWIF/SCIF.

File: lib/airtable.js (hunk 1 above)
Comment (saralain88) [reply lib/airtable.js:11]: It sure should, that cache does nothing right now. Good eye!

File: lib/options.js (hunk 2 above)
Comment (saralain88) [reply lib/options.js:47]: Imported and used in badge-notifier to show a "this is a test" warning: https://github.com/IdolFest/idolfest-discord/pull/4/files#diff-fd7913345d10ec1d693fbe5fd23dce5e935dc25e740d85c0a1ae78242c4ff25bR52

**Review by stevestreza — APPROVED** [review]

**Pull request merged**